packages/admin/cms-admin/src/common/MasterMenuRoutes.tsx

```diff
@@ -2,12 +2,14 @@ import { RouteWithErrorBoundary } from "@comet/admin";
 import * as React from "react";
 import { Redirect, RouteProps, Switch, useRouteMatch } from "react-router-dom";
 
+import { useCurrentUser } from "../userPermissions/hooks/currentUser";
 import { MasterMenuData, MasterMenuItem } from "./MasterMenu";
 
 export function getRoutePropsFromMasterMenuData(items: MasterMenuData): RouteProps[] {
-    // TODO: Filter for user-permissions once they are available
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I don't think you should disable the lint rule.

Either pass the user, or rename the function into a hook